info/v1/container.go

```diff
@@ -306,6 +306,11 @@ type MemoryStats struct {
 	// Units: Bytes.
 	Usage uint64 `json:"usage"`
 
+	// Various statistics reported in memory.stat in cgroupfs. See the section
```

Comment:
Yeah. Let's limit to RSS for now. We get quite a few questions/issues around the data exposed by cAdvisor. Honestly, all the data that cAdvisor exposes is documented as part of kernel APIs.